src/client/serialize/SConscript

```diff
@@ -0,0 +1,29 @@
+"""Build FUSE client"""
+import daos_build
+
+def scons():
+    """Execute build"""
+    Import('env')
+
+    denv = env.Clone()
+
+    have_hdf5 = True
+
+    #If hdf5 installed build hdf_serialize library
+    conf = Configure(denv)
+    if not conf.CheckLibWithHeader('hdf5', 'hdf5.h', 'c'):
+        have_hdf5 = False
+    conf.Finish()
+
+    libraries = ['daos', 'hdf5']
+
+    src = ['hdf_serialize.c']
+    if have_hdf5 is True:
+        print("Building with hdf5 bindings.")
+        hdf_serialize = daos_build.library(denv, 'hdf_serialize', src,
+                                           LIBS=libraries)
```

Comment:
The library name should help to recognize that this is daos-related to avoid confusion.